src/test_driver/linux-cirque/OnOffClusterTest.sh

```diff
@@ -29,9 +29,12 @@ function build_chip_tool() {
     # These files should be successfully compiled elsewhere.
     source "$REPO_DIR/scripts/activate.sh" >/dev/null
     set -x
+    gn gen out/debug >/dev/null
```

Comment:
Can we avoid building everything twice here.

It seems like chip-tool is important enough to just be added to the main build file, like chip-shell is. Add it here:
https://github.com/project-chip/connectedhomeip/blob/master/BUILD.gn#L70